### DISCUSSION TITLE
Unknown arg `guest` in data.guest for type QuestionUncheckedCreateInput

### DISCUSSION THREAD

**Discussion by Lukkyz** [prisma/prisma#19035] [Q&A]
### Bug description

Unknown arg `guest` in data.guest for type QuestionUncheckedCreateInput. Did you mean `guestId`? Available args:
type QuestionUncheckedCreateInput {
  id?: Int
  productId: Int
  question: String
  createdAt?: DateTime
  updatedAt?: DateTime
  userId?: Int | Null
  guestId?: Int | Null
  answers?: AnswerUncheckedCreateNestedManyWithoutQuestionInput
}

### How to reproduce

Run this query :
```js
await prisma.Question.create({
      data: {
        productId: id,
        question: "Hello world ?",
        guest: {
          create: { firstname: "John", lastname:"Doe", email: "example@example.com"}
        } 
      }
 })
```

### Expected behavior

I expect to not have error, and have guest successfully connected to question.

### Prisma information

```prisma

model Question {
  id        Int      @id @default(autoincrement())
  productId Int
  question  String   @db.VarChar(255)
  product   Products @relation(fields: [productId], references: [id])
  createdAt DateTime @default(now())
  updatedAt DateTime @default(now())
  user      Users?   @relation(fields: [userId], references: [id])
  guest     Guest?   @relation(fields: [guestId], references: [id])
  userId    Int?
  guestId   Int?
  answers   Answer[]
}

model Guest {
  id           Int            @id @default(autoincrement())
  email        String?        @db.VarChar(255)
  firstname    String?        @db.VarChar(255)
  lastname     String?        @db.VarChar(255)
  Question     Question[]
}
```

### Environment & setup

- OS
Debian 11
- Database 
MySQL
- Node.js
20.0.0

### Prisma Version

```
prisma 4.10
@prisma/client: ^4.10.0
```

**Comment by antipr000** [comment]
<img width="652" alt="Screenshot 2023-05-01 at 12 45 57 PM" src="https://user-images.githubusercontent.com/101579616/235421684-fa2e1a56-3e19-4914-93fd-8ab53d12ff49.png">

As per the generated files in `QuestionUncheckedCreateInput` it does not expect guest rather expects a guest id. One way to solve this is using `QuestionCreateInput` instead if you are okay with auto generating the `id` (which you are doing  already)

```
await prisma.Question.create({
      data: {
        product: {
          connect: {
             id
          }
        },
        question: "Hello world ?",
        guest: {
          create: { firstname: "John", lastname:"Doe", email: "example@example.com"}
        } 
      }
 })

**Reply by ludralph** [reply]
Hi @Lukkyz 

Did you get a chance to see the response from @antipr000? Did it help resolve your issue?

**Reply by ludralph** [reply]
Hi @Lukkyz 

Do you have any more questions around this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
